run-build.sh

```diff
@@ -18,11 +18,11 @@ if [[ ! -d $NETLIFY_REPO_DIR ]]; then
 fi
 cd $NETLIFY_REPO_DIR
 
-: ${NODE_VERSION="10"}
-: ${RUBY_VERSION="2.6.2"}
-: ${YARN_VERSION="1.13.0"}
+: ${NODE_VERSION="12.16.2"}
+: ${RUBY_VERSION="2.7.1"}
+: ${YARN_VERSION="1.22.4"}
 : ${PHP_VERSION="5.6"}
-: ${GO_VERSION="1.12"}
+: ${GO_VERSION="1.14.2"}
```

Comment:
nit: anyway to combine these version numbers with what the Dockerfile uses to reduce duplication? Minor concern though if complicated

Reply:
These versions are here as examples for the user to change if they want a different version in their test build. I'm not sure how we could deduplicate these without making this process more confusing to end users. Note that this script in this repo is only used for debugging by end users.